src/artemis/zocalo_interaction.py

```diff
@@ -0,0 +1,47 @@
+import zocalo.configuration
+from workflows.transport import lookup, default_transport
+import getpass
+import socket
+
+
+def _send_to_zocalo(parameters: dict):
+    zc = zocalo.configuration.from_file()
+    zc.activate()
+
+    transport = lookup(default_transport)()
+    transport.connect()
+    message = {
+        "recipes": "mimas",
+        "parameters": parameters,
+    }
+    header = {
+        "zocalo.go.user": getpass.getuser(),
+        "zocalo.go.host": socket.gethostname(),
+    }
+    transport.send("processing_recipe", message, headers=header)
```

Comment:
should this have transport.disconnect() at the end? (also feels like a disconnect should go in a finally block, although zocalo has it in the try...)